init.m

```diff
@@ -34,6 +35,10 @@
 Config.GRAVITY_ACC = [0,0,-9.81];
 Config.tStep = 0.001;
 
+% Use qpOASES instead of quadprog, typically in the case where the optimization toolbox is not
+% available.
+Config.USE_OSQP = true;
```

Comment:
Super minor comment: use qpOASES maybe is a leftover

Reply:
absolutelly, thanks!

Reply:
Fixed.